content/applications/services/helpdesk/overview/ratings.rst

```diff
@@ -49,12 +51,12 @@ moved to that stage. Customers will be asked to rate the support they received w
    :alt: View of a standard helpdesk customer review email template for Odoo Helpdesk.
 
 After selecting a rating, customers are taken to a webpage where they can provide specific written
-feedback to support their rating. Once a rating is submitted, it is added to the chatter on the
-ticket.
+feedback to support their rating. The rating is then submitted, and the rating, as well as any
+additional comments, are added to the chatter on the ticket.
 
 .. tip::
    Customer ratings can also be viewed through the :guilabel:`Customer Ratings` report. To view
-   this report, go to :menuselection:`Helpdesk --> Reporting --> Customer Ratings`.
+   this report, go to :menuselection:`Helpdesk app --> Reporting --> Customer Ratings`.
 
 .. seealso::
    :doc:`/applications/general/companies/email_template`
```

Comment:
After enabling the Customer Ratings setting, an option to publish ratings on the team’s website appears. Enabling this setting provides portal users with an overview of the ratings the team has received over the last thirty days. Specific written feedback will not be included; only statistics of the team’s performance will be visible.


Where does the option appear?